pkg/cmd/kafka/topic/describe/describe.go

```diff
@@ -88,6 +80,12 @@ func NewDescribeTopicCommand(f *factory.Factory) *cobra.Command {
 
 	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", opts.localizer.MustLocalize("kafka.topic.common.flag.output.description"))
 
+	cmd.Flags().StringVar(&opts.name, "name", "", opts.localizer.MustLocalize("kafka.topic.common.flag.output.description"))
+	_ = cmd.RegisterFlagCompletionFunc("name", func(cmd *cobra.Command, _ []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return cmdutil.FilterValidTopicNameArgs(f, toComplete)
+	})
+	_ = cmd.MarkFlagRequired("name")
```

Comment:
[Offtopic] It might be good to have our own helper for that that provides i18n support. 
I have been manually validating flags that are required due to lack of i18n support for `_ = cmd.MarkFlagRequired("name")`

Reply:
I don't follow. Something like `cmd.MarkFlagRequired(opts.localizer.MustLocalize("flag.id")`